gemini/prompts/examples/question_answering.ipynb

```diff
@@ -0,0 +1,943 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    question = "What machined are required for hosting Vertex AI models?"
</pre><p>What machines → spelling correction</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/478/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>